examples/trials/efficientnet/config_lr.yml

```diff
@@ -0,0 +1,24 @@
+authorName: v-yugzh
+experimentName: lr_efficient_net
+trialConcurrency: 4
+maxExecDuration: 48h
+maxTrialNum: 140
+trainingServicePlatform: pai
+searchSpacePath: search_lr.json
+useAnnotation: false
+tuner:
+  builtinTunerName: GridSearch
+trial:
+  codeDir: EfficientNet-PyTorch
+  command: python3 main.py cifar10 -j 4 --epochs 20 -a efficientnet-b0 --pretrained --num-classes 10 --batch-size 64 --optimizer sgd --request-from-nni
+  cpuNum: 4
+  memoryMB: 16384
+  gpuNum: 1
+  shmMB: 8192
+  virtualCluster: nni
+  image: ultmaster/nni:latest
```

Comment:
could you use the image from nni?

Reply:
In that case, PyTorch in NNI official image needs to be upgraded to at least v1.1; and nfs-common and cifs-utils need to be installed. Refer to [Dockerfile](https://github.com/ultmaster/nni-dockerfile/blob/master/Dockerfile) for all the changes I've done to make it work.